packages/core/src/user-store/sqlite/store.ts

```diff
@@ -438,8 +440,6 @@ export class SqliteUserStore implements UserStore {
 
     if (skip) {
       query = query.offset(skip);
-      // SQLite doesn't support OFFSET without LIMIT, so we need to set a limit.
-      if (!first) query = query.limit(-1);
```

Comment:
I guess this is unnecessary because after `parseModelFilter`, `first` will always be defined?

Reply:
yup